src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-attributes-edit.js

```diff
@@ -0,0 +1,171 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import AlloyEditor from 'alloyeditor';
+import EzWidgetButton from '../base/ez-widgetbutton';
+
+export default class EzBtnAttributesEdit extends EzWidgetButton {
+    constructor(props) {
+        super(props);
+
+        this.setDefaultAttributesMethods = {
+            tr: this.setDefaultAttributesOnTableRows,
+            td: this.setDefaultAttributesOnTableCells,
+            li: this.setDefaultAttributesOnListItems,
+        };
+
+        this.setDefaultClassesMethods = {
+            tr: this.setDefaultClassesOnTableRows,
+            td: this.setDefaultClassesOnTableCells,
+            li: this.setDefaultClassesOnListItems,
+        };
+    }
+    componentDidMount() {
+        const block = this.findSelectedBlock();
+
+        this.setDefaultClasses(block);
+        this.setDefaultAttributes(block);
+    }
+
+    setDefaultClasses(block) {
+        if (!Object.keys(this.classes).length || block.$.classList.contains('ez-classes-added') || !this.classes.defaultValue) {
+            return;
+        }
+
+        const defaultValue = this.classes.defaultValue.split(',');
+        const setDefaultClassesMethod = this.setDefaultClassesMethods[this.toolbarName]
```

Comment:
I don't like the idea of having very long methods (with the switch it would be very long). When I'm checking what method is doing and looking for a specific part of the code it's easier to check short method and go another one than reading very long method and analyzing which part of the code is important for me in this case.